### PR TITLE
Documentation: src/speed-test.js, src/object-registry.js, src/grid-api-client.js

### PR DESCRIPTION
**Pull request by thefirebanks**
## Description
Addresses issues #124 and #128, adds inline documentation to `SpeedTest` and 'GridAPIClient`. Also fixes small documentation issues in `ObjectRegistry`.

## Affected Dependencies
N/A

## How has this been tested?
N/A

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/object-registry.js [hunk 1]
```diff
@@ -2,19 +2,20 @@ import * as tf from '@tensorflow/tfjs-core';
...
  * ObjectRegistry stores a map of {id -> object} and a map of {id -> gc},
- * where gc denotes if we want the object of a given id to be garbage collected.
- *
- * @property {Object.<string, Plan>} objects
- * @property {Object.<string, Protocol>} gc
+ * where gc denotes if the object of given id will be garbage collected.
  */
 export default class ObjectRegistry {
+  /**
+   * @property {Object.<string, Object>} objects - a map of {id: object}
```
Comment (vvmnnnkv) [inline src/object-registry.js:9]: Hmm, why this is moved into constructor's docblock?

File: src/object-registry.js [hunk 2]
```diff
@@ -46,7 +47,7 @@ export default class ObjectRegistry {
   }
 
   /**
-   * Load the objects from an existing ObjectRegistry and store them in this.objects
+   * Loads the objects from an existing ObjectRegistry and store them in this.objects
```
Comment (vvmnnnkv) [inline src/object-registry.js]: Then store -> stores? :-)

File: src/speed-test.js [hunk 3]
```diff
@@ -1,6 +1,20 @@
 import { createRandomBuffer } from './utils/random-buffer';
 
+/**
+ * SpeedTest is a class that contains the necessary components
+ * to measure download/upload speed, and ping.
+ */
 export class SpeedTest {
+  /**
+   * @property {string} downloadUrl
```
Comment (vvmnnnkv) [inline src/speed-test.js]: Shouldn't this be @param (constructor's argument)?

**Comment by vvmnnnkv** [comment]
A bit confused about @property usage inside constructor's docblock, otherwise looks great!

File: src/object-registry.js (hunk 1 above)
Comment (thefirebanks) [reply src/object-registry.js:9]: I saw Nahua's latest documentation PR and thought that it was the convention? But I can take it back if needed

File: src/object-registry.js (hunk 2 above)
Comment (thefirebanks) [reply src/object-registry.js]: Whoooooops

File: src/speed-test.js (hunk 3 above)
Comment (thefirebanks) [reply src/speed-test.js]: Absolutely, thanks! Should I leave them as parameters or move them up to the class doc as properties?

Comment (thefirebanks) [reply src/speed-test.js]: Left them as parameters!

**Review by vvmnnnkv — APPROVED** [review]

**Comment by vvmnnnkv** [comment]
Nice work! ✨

**Pull request merged**